cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java

```diff
@@ -83,6 +83,12 @@ public final class DataprocUtils {
   // This property needs to be overridden in cdap-site.
   public static final String TROUBLESHOOTING_DOCS_URL_DEFAULT = "";
 
+  // resources required by driver (default runtime job) running on driver pool node
+  public static final String DRIVER_MEMORY_MB = "driverMemoryMB";
+  public static final String DRIVER_MEMORY_MB_DEFAULT = "2048";
+  public static final String DRIVER_VCORES = "driverVCores";
+  public static final String DRIVER_VCORES_DEFAULT = "2";
```

Comment:
Do we really need 2 cores for the runtime job?

Reply:
These numbers can be tuned after some performance tests.